File: Examen  Final-Doctor Paciente/Final/app/src/main/java/com/example/afinal/Paciente.kt
package com.example.afinal

class Paciente
    (var id: Int,var nombre:String,
               var edad:String, var tipoSangre: String,
               var cedula: String, var doctor: String,
var latitud: String, var longitud: String,var url:String, var urlImagen: String) {


    override fun toString(): String {
        return "id: ${id}, nombre:${nombre}, edad: ${edad}, tipo de Sangre:${tipoSangre}, doctor:${doctor}"
    }


}

File: Examen  Final/Back/hospital/api/models/Paciente.js
/**
 * Paciente.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

     nombre:{
    type: 'string',
  },
  edad:{
    type: 'string',
  },
  tipoSangre:{
    type: 'string',

  }, cedula:{
    type: 'string',

  },

  doctor:{
    type: 'string',

  },
  latitud :{
    type: 'string',

  },
  longitud:{
    type: 'string',

  },
  url:{
    type: 'string',

  },
  urlImagen:{
    type: 'string',

  },

  },

};

